cpp/src/utils/sockaddr.cpp

```diff
@@ -51,13 +51,16 @@ void sockaddr_get_ip_port_str(const struct sockaddr_storage* sockaddr,
       addr_in = reinterpret_cast<decltype(addr_in)>(sockaddr);
       inet_ntop(AF_INET, &addr_in->sin_addr, ip_str, max_str_size);
       snprintf(port_str, max_str_size, "%u", ntohs(addr_in->sin_port));
+      break;
     case AF_INET6:
       addr_in6 = reinterpret_cast<decltype(addr_in6)>(sockaddr);
       inet_ntop(AF_INET6, &addr_in6->sin6_addr, ip_str, max_str_size);
       snprintf(port_str, max_str_size, "%u", ntohs(addr_in6->sin6_port));
+      break;
```

Comment:
Without the `break` we were always falling through to the `default` branch, but now it _does_ something, so we were getting nonsense ip and ports.